### PR TITLE
Update angular to 1.2.10

### PR DESCRIPTION
**Pull request by julesbou**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -8,10 +8,12 @@
...
   "dependencies": {
-    "angular": "~1.0.7",
-    "angular-resource": "~1.0.7",
-    "angular-mocks": "~1.0.7",
-    "angular-scenario": "~1.0.7",
+    "angular": "~1.2.7",
+    "angular-resource": "latest",
+    "angular-ui-router": "latest",
+    "angular-mocks": "latest",
+    "angular-mocks": "latest",
```
Comment (dmitryevseev) [inline bower.json]: duplicate

File: bower.json [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
     "node_modules",
     "components"
   ],
   "dependencies": {
-    "angular": "~1.0.7",
-    "angular-resource": "~1.0.7",
-    "angular-mocks": "~1.0.7",
-    "angular-scenario": "~1.0.7",
+    "angular": "~1.2.8",
+    "angular-ui-router": "~0.2.8",
```
Comment (julesbou) [inline bower.json]: @dmitryevseev when doing a `bower install` there's some files missing for "angular-ui-router". Until this problem is fixed, this PR cannot be merged of course :)

Comment (dmitryevseev) [inline bower.json]: I've discussed with Iwein and Tiziano already idea about disposing deps to be checked into the repo. Just having a strict versions numbers in bower.json will be enough - going to do that.

**Comment by julesbou** [comment]
@dmitryevseev finally updated to 1.2.10 instead of 1.2.7

**Pull request merged**